inc/assets/js/parts/_main_side_nav.part.js

```diff
@@ -0,0 +1,162 @@
+var czrapp = czrapp || {};
+/************************************************
+* SIDE NAV SUB CLASS
+*************************************************/
+(function($, czrapp) {
+  var _methods =  {
+    init : function() {
+      this.$_sidenav                = $( '#tc-sn' );
+      
+      if ( ! this._is_sn_on() )
+        return;
+
+      //cache jQuery el
+      this.$_page_wrapper           = $('#tc-page-wrap');
+      this.$_page_wrapper_node      = this.$_page_wrapper.get(0);
+      
+      this.$_sidenav_inner          = $( '.tc-sn-inner', this.$_sidenav);
+      
+      this._toggle_event            = czrapp.$_body.hasClass('tc-is-mobile') ? 'touchstart' : 'click';
+
+      this._browser_can_translate3d = ! czrapp.$_html.hasClass('no-csstransforms3d');
+
+      /* Cross browser support for CSS "transition end" event */
+      this.transitionEnd            = 'transitionend webkitTransitionEnd otransitionend oTransitionEnd MSTransitionEnd';
+
+      //fire event listener
+      this.sideNavEventListener();
+      
+      this._set_offset_height();
+
+    },//init()
+
+
+    sideNavEventListener : function() {
+      var self = this;
+      
+      //BUTTON CLICK/TAP
+      czrapp.$_body.on( this._toggle_event, '.sn-toggle', function( evt ) {
+          self.sideNavEventHandler( evt, 'toggle' );
+      });
+
+      //TRANSITION END
+      this.$_page_wrapper.on( this.transitionEnd, function( evt ) {
+          self.sideNavEventHandler( evt, 'transitionend' );
+      });
+
+      //RESIZING ACTIONS
+      czrapp.$_window.resize( function( evt ) {
+        self.sideNavEventHandler( evt, 'resize');
+      });
+
+      czrapp.$_window.scroll( function( evt ) {
+        self.sideNavEventHandler( evt, 'scroll');
+      });
+    },
+
+
+
+    sideNavEventHandler : function( evt, evt_name ) {
+      var self = this;
+      
+      switch ( evt_name ) {
+        case 'toggle':
+          // prevent multiple firing of the click event
+          if ( ! this._is_translating() )
+            this._toggle_callback( evt );
```

Comment:
Expected '{' and instead saw 'this'.<br>Identifier '_toggle_callback' is not in camel case.